### PR TITLE
Covers our ass with the ToS by renaming the licorice creamsicle from being a specific product reference to generic creamsicles.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Covers our ass with the ToS by renaming the licorice creamsicle from being a specific product reference to generic creamsicles.

## Why It's Good For The Game

It's good for the game for us to have a github repository still, and specific references to very poorly chosen names for products is not a good idea for keeping our repository up and running. SS14 pointed this out during a PR at https://github.com/space-wizards/space-station-14/pull/8684

## Changelog
:cl:
spellcheck: Nanotrasen has lost the rights to several popular confectionaries, and has created "original" replacements.
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
average ss13 player humor

**Comment by Putnam3145** [comment]
at some point we should probably just grep every `gg` in ss13 and replace it with `**`

File: code/game/objects/items/food/frozen.dm [hunk 1]
```diff
@@ -292,12 +292,12 @@
 	food_reagents = list(/datum/reagent/consumable/hot_coco = 4, /datum/reagent/consumable/cream = 2, /datum/reagent/consumable/vanilla = 3, /datum/reagent/consumable/sugar = 2)
 	overlay_state = "jumbo"
 
-/obj/item/food/popsicle/nogga_black
-	name = "nogga black"
-	desc = "A salty licorice icecream recently reintroduced due to all records of the controversy being lost to time. Those who cannot remember the past are doomed to repeat it."
+/obj/item/food/popsicle/licorice_creamsicle
+	name = "licorice creamsicle"
```
Comment (optimumtact) [inline code/game/objects/items/food/frozen.dm:296]: can we come up with our own name

Comment (Iamgoofball) [reply code/game/objects/items/food/frozen.dm:296]: i'll poll the players

**Comment by BlueMemesauce** [comment]
"Generic creamsicles" is still a specific product reference, Creamsicle® is a registered trademark of Good Humor-Breyers

**Comment by necromanceranne** [comment]
Might wanna just ask the common core folk for an idea maybe.

My suggestion? 

"Spacer's Delight licorice cream treat"

**Comment by EOBGames** [comment]
I'm gonna throw my hat into the ring and suggest either "Void Bar", "Black Hole", or "Milky Way".

**Comment by necromanceranne** [comment]
Milky Way is a thing.

https://en.m.wikipedia.org/wiki/Milky_Way_(chocolate_bar)

**Comment by Krysonism** [comment]
Before this is reflexively removed i ought to provide some context regarding this item.

This is in no way a ToS violation, rather it is a commentary upon a culturally significant event in my country; the nogger black controversy of 2005.

My explanation from discord:

> I wanted to create a few popsicles with bitten states as I enjoy making sweets in game.
> 
> After making two generic creamsicles and parodies of the omnipresent European magnum & Cornetto ice creams. I could not pass up making a parody of the short lived but legendary nogger black, a salty licorice spin on the old nogger nougat ice cream.
> 
> In my county of Sweden, nogger black became a sort of proto meme shortly after its launch.
> 
> The "nogger black controversy" of 2005 saw the country divided into camps,  on one side, we had the non profit "centrum against racism" who declared that the name was horrendously racist.
> 
> On the other side we had the giant ice cream conglomerate GB glace who claimed the ice cream was falling victim to a smear campaign.
> 
> Much focus was spent on the alleged graffiti styling on the ice cream wrapper and its implications.
> 
> Perhaps no ice cream in history has had so much cultural impact or sparked so much conversation, it would be a shame if it was neutered just because some Americans are unaware of the context.

bild

**Comment by EOBGames** [comment]
> Milky Way is a thing.
> 
> https://en.m.wikipedia.org/wiki/Milky_Way_(chocolate_bar)

I'm aware, it's one of my favourite chocolate bars- but the issue here (as best I can tell, given the specifics of what's being targeted) is not that it's a product reference despite what the title says (as half our food items are specific product references), but the name being dodgy around the ToS.

**Comment by necromanceranne** [comment]
True, just wanna prevent any confusion is all

**Comment by Farquaar** [comment]
I don't mind the change but you really should wait until your PR gets merged before you start changing the food's name on the wiki.

**Comment by Mothblocks** [comment]
names kinda dumb but im not hung up

**Comment by Twaticus** [comment]
> I don't mind the change but you really should wait until your PR gets merged before you start changing the food's name on the wiki. image

holy shit lol

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm [hunk 2]
```diff
@@ -297,8 +297,8 @@
 	result = /obj/item/food/popsicle/jumbo
 	subcategory = CAT_ICE
 
-/datum/crafting_recipe/food/nogga_black
-	name = "Nogga black"
+/datum/crafting_recipe/food/licorice_creamsicle
+	name = "Licorice popsicle"
```
Comment (Dorsisdwarf) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm:301]: Is this intentional?

Comment (Iamgoofball) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm:301]: whoops, forgot to rename to Void Bar :tm: there

**Pull request merged**